src/store/actions.js

```diff
@@ -95,13 +99,7 @@ export default {
           names = flatten(names);
           names = uniqBy(names, 'name');
           if (!process.env.RUNNING_IN_TESTS) {
-            if (names.length) {
-              commit(types.SET_ACCOUNT_AENS, {
-                account: index,
-                aename: names[0].name,
-                pending: !!names[0].pending,
-              });
-            } else {
```

Comment:
Maybe we should leave this, because when imported my account and have registered .chain names I saw message `Claim your .chain name`

Reply:
> Maybe we should leave this, because when imported my account and have registered .chain names I saw message `Claim your .chain name`

Agree.

Reply:
This code does not work properly. It always takes the first name. 
I suggest if the user has registered name, the `Claim your .chain name` message to be changed to `Set your active name` or something like that. Or we can make the user to activate his name after registration.

Reply:
> This code does not work properly. It always takes the first name.
> I suggest if the user has registered name, the `Claim your .chain name` message to be changed to `Set your active name` or something like that. Or we can make the user to activate his name after registration.

This is also a good proposal. Please do it.